### ISSUE TITLE
Mangakalot download error

### ISSUE THREAD

**Issue by donotos** [fmd-project-team/FMD#972]
Hello.

Since yesterday FMD connot download from the mangakalot sites (mangakalot, mangakalots, manganelo) all my downloads end up in error.

**Comment by Rarksoca** [comment]
same here

**Comment by webanimax** [comment]
Reporting, same here.

**Comment by yagamiyakumo** [comment]
Same there

**Comment by spamtaztik** [comment]
Manganelo is working fine now with my FMD. I don't know about the other sites though.

**Comment by admiralmikan** [comment]
Yes, even after adding cookie on Manganelo config, it's hit and miss, some titles can't be downloaded. Mangakakalot got same problem.

**Comment by Rarksoca** [comment]
Same here, i can't download some chapter.
I guess it is because manganelo.com site problem.

<img src="https://i.paste.pics/14e2cb3153e3788bdaa8f3fdac3a6471.png" />

**Comment by millenopan** [comment]
I seem to still be having this issue, where certain chapters do not download.

**Comment by Tmp341** [comment]
Are those chapters accessible via browser?

**Comment by millenopan** [comment]
Yeah, I tried opening them directly from a browser and it was loading fine, so I'm not sure what seems to be the problem.

**Comment by yagamiyakumo** [comment]
Facing the same issues. Most chapters besides the recently uploaded ones can't be downloaded. I can view them just fine on my PC but not on my phone. I wonder if there's some re-directing issues?
